pkgs/tools/security/pcsclite/default.nix

```diff
@@ -34,14 +35,12 @@ stdenv.mkDerivation rec {
     "--enable-confdir=/etc"
     # The OS should care on preparing the drivers into this location
     "--enable-usbdropdir=/var/lib/pcsc/drivers"
-  ]
-  ++ (if stdenv.isLinux then [
+    (lib.enableFeature stdenv.isLinux "systemd")
+    (lib.enableFeature polkitSupport "polkit")
+  ] ++ lib.optionals stdenv.isLinux [
     "--enable-ipcdir=/run/pcscd"
-    "--enable-polkit"
     "--with-systemdsystemunitdir=${placeholder "bin"}/lib/systemd/system"
-  ] else [
-    "--disable-libsystemd"
```

Comment:
Isn't this deletion going to break Darwin builds?

Reply:
Notice the `lib.enableFeature stdenv.isLinux "systemd"` above: it should be equivalent but cleaner..

Reply:
Ah, I see.